packages/DataGrid/src/components/Cell/Cell.js

```diff
@@ -41,18 +48,47 @@ export default function Cell(props) {
 
   const cellProps = typeof column.cellProps === "function" ? column.cellProps(options) : {};
 
+  function setHighlight(value = false) {
+    setIsHighlighted(value);
+  }
+
+  React.useEffect(() => {
+    const key = getGridRefId({ gridId, rowIndex, columnIndex });
+    window.paprika.dataGridRef[key] = {
+      setHighlight,
+    };
+
+    return () => {
+      console.log("before:", Object.keys(window.paprika.dataGridRef).length);
+      delete window.paprika.dataGridRef[key];
+      console.log("after:", Object.keys(window.paprika.dataGridRef).length);
+    };
+  }, []); // eslint-disable-line
+
+  const isColumnCellAFunction = typeof column.cell === "function";
   return (
     <sc.Cell
       borderType={borderType}
-      data-cell={`${gridId}.${columnIndex}.${rowIndex}`}
+      data-pka-cell-key-column-index={columnIndex}
+      data-pka-cell-key-row-index={rowIndex}
+      data-pka-cell-key={`${gridId}.${columnIndex}.${rowIndex}`}
+      data-pka="cell-container"
```

Comment:
Should be `data-pka-anchor='data-grid.cell-container'` unless there's a reason to deviate from the standard?